packages/twenty-front/src/modules/object-metadata/utils/getAvatarIcon.ts

```diff
@@ -0,0 +1,35 @@
+import { useTheme } from '@emotion/react';
+import { IconCheckbox, IconComponent, IconNotes } from 'twenty-ui';
+
+export const GetAvatarIcon = (objectNameSingular: string) => {
```

Comment:
Should this be a hook or a util?
It works like a util but since it has usetheme in it it needs to be a React component or hook.
I was confused so for now I kept it in util folder with React naming convention :(

Reply:
I'm surprised that the linter let you call a hook in a util. This should be a hook exporting utils : `useGetStandardObjectIcon`